central/processlisteningonport/datastore/datastore_impl_test.go

```diff
@@ -0,0 +1,1149 @@
+package datastore
+
+import (
+	"context"
+	"testing"
+	"time"
+
+	"github.com/golang/mock/gomock"
+	processIndicatorDataStore "github.com/stackrox/rox/central/processindicator/datastore"
+	processIndicatorSearch "github.com/stackrox/rox/central/processindicator/search"
+	processIndicatorStorage "github.com/stackrox/rox/central/processindicator/store/postgres"
+	plopStore "github.com/stackrox/rox/central/processlisteningonport/store/postgres"
+	postgresStore "github.com/stackrox/rox/central/processlisteningonport/store/postgres"
+	"github.com/stackrox/rox/central/role/resources"
+	"github.com/stackrox/rox/generated/storage"
+	"github.com/stackrox/rox/pkg/env"
+	"github.com/stackrox/rox/pkg/fixtures/fixtureconsts"
+	"github.com/stackrox/rox/pkg/postgres/pgtest"
+	"github.com/stackrox/rox/pkg/protoconv"
+	"github.com/stackrox/rox/pkg/sac"
+	"github.com/stretchr/testify/suite"
+)
+
+func TestPLOPDataStore(t *testing.T) {
+	suite.Run(t, new(PLOPDataStoreTestSuite))
+}
+
+type PLOPDataStoreTestSuite struct {
+	suite.Suite
+	datastore          DataStore
+	store              plopStore.Store
+	indicatorDataStore processIndicatorDataStore.DataStore
+
+	postgres *pgtest.TestPostgres
+
+	hasNoneCtx  context.Context
+	hasReadCtx  context.Context
+	hasWriteCtx context.Context
+
+	mockCtrl *gomock.Controller
+}
+
+func (suite *PLOPDataStoreTestSuite) SetupTest() {
+	if !env.PostgresDatastoreEnabled.BooleanSetting() {
```

Comment:
if we use a `SetupSuite` we should be able to do this check one time and then call a `suite.T().Skip()` or something like that and skip the entire suite so we don't have to put this check in each test.  Something like in `central/node/datastore/dackbox/datastore/datastore_impl_test.go` only the reverse case.

Reply:
Done